tests/e2e/support/api/share/share.ts

```diff
@@ -41,3 +43,44 @@ export const createShare = async ({
   })
   checkResponseStatus(response, 'Failed while creating share')
 }
+
+export const acceptShare = async ({ user, path }: { user: User; path: string }): Promise<void> => {
+  const params = 'include_tags=false&state=all&shared_with_me=true'
+  const shareId = await getShareId({ user, path, params })
```

Comment:
[URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) can be utilized here. makes it more readable.
```suggestion
  const params = new URLSearchParams({
      include_tags: false,
      state: 'all',
      shared_with_me: true
  })
  const shareId = await getShareId({ user, path, params.toString() })
```